Homework 1.py

```diff
@@ -0,0 +1,371 @@
+import csv
+import glob
+import itertools
+from itertools import groupby
+import random
+import re
+import sys
+from time import localtime
+import unittest
+import xml.etree.ElementTree as etree
+
+# 1 line: Output
```

Comment:
It looks much more clear if your task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
   pass  # last line of the previous task
    
 
   # next task docstring
   pass  # my next task first line  
```